modules/ROOT/pages/promote-api.adoc

```diff
@@ -58,6 +58,11 @@ To promote an API:
 * *Client ID enforcement via headers*: Use if you authenticate to your API passing `client_id` and `client_secret` headers.
 * *Client ID enforcement via query parameters*: Use if you authenticate to your API passing `client_id` and `client_secret` query params.
 * *Custom*: Use if you authenticate to your API using custom header parameters and values.
+. To configure mutual authentication (mTLS) between the API and DataGraph, select *Configure TLS/SSL* and provide the following:
+*  A public certificate signed by your organization's “root” certificate authority (CA)
+* The private key used to generate the certificate signing request (CSR)
+* The password for the encrypted private key
+* Optionally, the root CA
```

Comment:
Same as add api. i would also add that if the user is promoting an api that is present on the target env, we will check if on that env the api had an already existing keystore or truststore. then the user will have the option to keep that keystore/truststore, or remove it, or remove it and add a new one. The user will not be able to automatically send the keystore/truststore present on the source env, it will need to remove the existing one (if present) and add the keystore/truststore as if it did not exist before.